superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -654,6 +655,7 @@ const DashboardBuilder: FC<DashboardBuilderProps> = () => {
               return (
                 <FiltersPanel
                   width={filterBarWidth}
+                  hidden={isReport}
```

Comment:
Do we still need `hidden` in line 670?

Reply:
The prop itself it is still required for the component as it can be included directly like is the case of the horizontal FilterBar. In this specific case, as the wrapper is hidden too, it's not necessary to pass it to the FilterBar too most probably but I think there is no harm in having it there either

Reply:
Let me quickly test and remove it if it causes no harm

Reply:
Removed. Looks good